<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchSearcher</class>
 <widget class="QDialog" name="BatchSearcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批量选股</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>302</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblStartDate">
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>开始日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dteStartDate">
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblEndDate">
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>结束日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dteEndDate">
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblInterval">
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>选股间隔</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbInterval">
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbbIntervalType">
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lblCriteriaName">
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>选股条件名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="iptCriteriaName">
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStartSearch">
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="text">
         <string>开始选股</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnStartSearch</sender>
   <signal>clicked()</signal>
   <receiver>BatchSearcher</receiver>
   <slot>start_searching()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start_searching()</slot>
 </slots>
</ui>
